jme3-core/src/main/java/com/jme3/renderer/opengl/GLRenderer.java

```diff
@@ -2995,6 +3034,9 @@ public void setVertexAttrib(VertexBuffer vb, VertexBuffer idb) {
                 attribs[slot] = vb.getWeakRef();
             }
         }
+        if (debug && caps.contains(Caps.GLDebug)) {
+            if (vb.getName() != null) glext.glObjectLabel(GLExt.GL_BUFFER, vb.getId(), vb.getName());
```

Comment:
It should suffice to do this once right after creating the GL object (glGenBuffers). Then you also have less chance of missing something, e.g. index buffers.